packages/cli/commands/features.ts

```diff
@@ -46,13 +52,11 @@ export const createFeatureAction = async (
       });
     }
 
-    spinner = ora(
-      `Creating feature for app ${chalk.cyan(appId)} at ${chalk.cyan(baseUrl)}...`,
-    ).start();
+    spinner = ora(`Creating feature...`).start();
     const feature = await createFeature(appId, name, key);
     // todo: would like to link to feature here but we don't have the env id, only app id
     spinner.succeed(
-      `Created feature ${chalk.cyan(feature.name)} with key ${chalk.cyan(feature.key)} at ${chalk.cyan(baseUrl)}.`,
+      `Created feature ${chalk.cyan(feature.name)} with key ${chalk.cyan(feature.key)}${baseUrlSuffix(baseUrl)}.`,
```

Comment:
can we link to the feature on bucket?

Reply:
Not without envId or going back to appId in the URL (which I think is the plan long term)

Reply:
Fixed